packages/jellyfish-api-core/__tests__/category/misc/deriveAddresses.test.ts

```diff
@@ -0,0 +1,58 @@
+import { RegTestContainer } from '@defichain/testcontainers/dist/index'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '../../../src'
+
+describe('derive addresses', () => {
+  const container = new RegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should derive an address without range', async () => {
+    const descriptor = 'wpkh(tprv8ZgxMBicQKsPd7Uf69XL1XwhmjHopUGep8GuEiJDZmbQz6o58LninorQAfcKZWARbtRtfnLcJ5MQ2AtHcQJCCRUcMRvmDUjyEmNUWwx8UbK/1/1/0)#t6wfjs64'
+    const address = ['bcrt1qjqmxmkpmxt80xz4y3746zgt0q3u3ferr34acd5']
+    expect(await client.misc.deriveAddresses(descriptor)).toStrictEqual(address)
```

Comment:
You should also check for the exact error thrown.

```suggestion
    const promise = client.misc.deriveAddresses(descriptor)).toStrictEqual(address)
    await expect(promise).rejects.toThrow(RpcApiError)
    await expect(promise).rejects.toMatchObject({
      payload: {
        code: -5,
        message: '<err message>',
        method: 'deriveaddresses'
      }
    })
```